backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleExecutor.java

```diff
@@ -190,14 +190,14 @@ public AnsibleReturnValue runCommand(AnsibleCommandConfig commandConfig, int tim
     }
 
     private AuditLogable createAuditLogable(AnsibleCommandConfig command, String taskName) {
-        if (command.hosts() == null) {
+        if (command.host() == null) {
```

Comment:
We can remove the check if we add a check to AnsibleExecutor.runCommand

Reply:
done